.github/workflows/main.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       if: ${{ matrix.numpy_ver != 'latest'}}
       run: |
         pip install --no-binary :numpy: numpy==${{ matrix.numpy_ver }}
+        pip install "xarray<2022.06"
```

Comment:
For a general solution, move this block before the "Install standard dependencies".  This forces the later install of pandas to be compliant.

Reply:
I think the xarray limit can be removed.  Testing at pysatNASA.  2022.09 should have fixed the bug on xarray's side.

Reply:
I updated the card in the project that has this.  https://github.com/orgs/pysat/projects/2

Reply:
The xarray limit fixes pandas.  Otherwise you need numpy 1.20.3

Reply:
I'm fine with whatever works.  I removed this at pysatNASA and it worked, as long as numpy is installed first.  The standard installs will then find a compliant pandas version, since we're not downgrading numpy afterward.  But xarray has some weird compliance issues.

Reply:
Using 1.20 only failed for me, you can check out the commit: https://github.com/pysat/pysatSpaceWeather/pull/94/commits/e8933550ec083c7ba527d0d419b6268a2396862e

Reply:
I think it's the order of operations that fixed things.  Check out https://github.com/pysat/pysatNASA/blob/cbad07654bfbb39c9b41f0bf726694945c872452/.github/workflows/main.yml

but mileage may vary, and there's another new xarray as of 8 days ago

Reply:
Having the block before or after worked, again you can see in the commits on this PR.  I only changed it because you suggested it and I liked the flow better.

Reply:
No problem.  I think I gave the wrong advice the first time around.  If it's working, go with it.